community.html

```diff
@@ -65,6 +65,18 @@ <h2>Join us</h2>
            </div>
          </div>
 
+         <div class="col-12 col-md-4">
+           <div class="card border-0 d-flex h-100">
+             <div class="card-header d-flex align-items-center justify-content-center h-100 border-0">
+               <a href="https://matrix.to/#/#riot-os:matrix.org" target="_blank"><img class="card-img-top" src="{{ "assets/img/matrix_logo.png" | relative_url }}" alt="Matrix logo"></a>
+             </div>
+             <div class="card-body px-0">
+               <!--<h4 class="card-title">Our matrix for discussions.</h4>-->
+               <a href="https://matrix.to/#/#riot-os:matrix.org" class="btn btn-outline-primary w-100" target="_blank">Join our community chat and ad-hoc discussions</a>
```

Comment:
s/Join our community chat and ad-hoc discussions/Join for ad-hoc discussions/

Reply:
```vimscript
s/Join our community chat and ad-hoc discussions/Join for ad-hoc discussions/<cr>
Git commit --amend<cr>
Git push --force-with-lease<cr>
```
;-)